airflow/dag_processing/collection.py

```diff
@@ -234,18 +233,15 @@ def _update_dag_warnings(
         session.merge(warning_to_add)
 
 
-def _update_import_errors(
-    files_parsed: set[str], import_errors: dict[str, str], processor_subdir: str | None, session: Session
-):
+def _update_import_errors(files_parsed: set[str], import_errors: dict[str, str], session: Session):
     from airflow.listeners.listener import get_listener_manager
 
     # We can remove anything from files parsed in this batch that doesn't have an error. We need to remove old
     # errors (i.e. from files that are removed) separately
 
     session.execute(delete(ParseImportError).where(ParseImportError.filename.in_(list(files_parsed))))
 
-    query = select(ParseImportError.filename).where(ParseImportError.processor_subdir == processor_subdir)
-    existing_import_error_files = set(session.scalars(query))
+    existing_import_error_files = set(session.scalars(select(ParseImportError.filename)))
```

Comment:
Would this need a "bundle id"? i.e. if there are 2 files with same names across bundles

Reply:
I'm simply trying to break out parts to make it easier to review :)